test/runtests.jl

```diff
@@ -0,0 +1,13 @@
+
+using Test
+
+using FiniteDiff
```

Comment:
CLSDyn uses `FiniteDifferences`. Maybe we should pick one?

Reply:
I think `FiniteDiff` is better than `FiniteDifferences`. And `FiniteDiff` is able to run on the GPU, I am not sure this is the case for `FiniteDifferences` :)